src/mpid/ch4/src/ch4_progress.h

```diff
@@ -133,6 +159,10 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_progress_test(MPID_Progress_state * state)
         for (int vci = 0; vci < MPIDI_global.n_vcis; vci++) {
             MPIDI_PROGRESS(vci);
         }
+        if (!made_progress && MPIR_CVAR_CH4_PROGRESS_THROTTLE && 
+            no_progress_counter > MPIR_CVAR_CH4_PROGRESS_THROTTLE_NO_PROGRESS_COUNT) {
+            usleep(1);
+        }
```

Comment:
I originally tried to use the same counter for global progress and throttle. Since you are using separate counters anyway, we might as well make global progress and throttle independent. How about move this code below -- line 183?

Reply:
Moved to 183 (in `else` during function exit when we fail to make progress)

Reply:
I also removed the change to the conditional in `MPIDI_do_global_progress` because we are keeping this separate from the global progress counter for now. I would like to study merging the counters further, but my priority is getting this workaround merged for Aurora.